src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -861,50 +861,58 @@ protected function doProcessRefundsWithoutIds($unsyncedRefundsWithoutId, $woocom
+                                'description' => sprintf(
+                                    __('Refund via Wordpress plugin (Refund #%s)', I18N::DOMAIN),
+                                    $refundId
+                                ),
+                            ],
                         ],
-                    ],
-                ]);
-                $this->debug('Storekeeper refund was created', [
-                    'payment' => $storekeeperPayment,
-                    'refund_id' => $storekeeperRefundId,
-                ]);
-                PaymentGateway::updateRefund($id, $storekeeperRefundId, $refundAmount);
-            } catch (GeneralException $generalException) {
-                if ('Only invoiced orders can be refunded' === $generalException->getMessage()) {
-                    $storekeeperRefundId = PaymentGateway::createRefundAsPayment($refundId, $refundAmount);
+                    ]);
                     $this->debug('Storekeeper refund was created', [
-                        'order_id' => $storekeeperId,
-                        'payment_id' => $storekeeperRefundId,
+                        'payment' => $storekeeperPayment,
+                        'refund_id' => $storekeeperRefundId,
                     ]);
                     PaymentGateway::updateRefund($id, $storekeeperRefundId, $refundAmount);
-                    $this->syncPaymentToBackend($storekeeperRefundId, $storekeeperId);
-                } else {
-                    throw $generalException;
+                } catch (GeneralException $generalException) {
+                    if ('Only invoiced orders can be refunded' === $generalException->getMessage()) {
+                        $storekeeperRefundId = PaymentGateway::createRefundAsPayment($refundId, $refundAmount);
+                        $this->debug('Storekeeper refund was created', [
+                            'order_id' => $storekeeperId,
+                            'payment_id' => $storekeeperRefundId,
+                        ]);
+                        PaymentGateway::updateRefund($id, $storekeeperRefundId, $refundAmount);
+                        $this->syncPaymentToBackend($storekeeperRefundId, $storekeeperId);
+                    } else {
+                        throw $generalException;
+                    }
                 }
+            } else {
+                $storekeeperRefundId = PaymentGateway::createRefundAsPayment($refundId, $refundAmount);
+                $this->debug('Storekeeper refund was created', [
+                    'order_id' => $storekeeperId,
+                    'payment_id' => $storekeeperRefundId,
+                ]);
+                PaymentGateway::updateRefund($id, $storekeeperRefundId, $refundAmount);
+                $this->syncPaymentToBackend($storekeeperRefundId, $storekeeperId);
             }
-
-            PaymentGateway::markRefundAsSynced($woocommerceOrderId, $storekeeperRefundId, $refundId);
```

Comment:
why was this line removed

Reply:
Both if and else calls `PaymentGateway::markRefundAsSynced($woocommerceOrderId, $storekeeperRefundId, $refundId);` and PHPStan flagged it so I moved it outside